.github/workflows/build.yml

```diff
@@ -208,9 +209,15 @@ jobs:
 
       - run: pipx run twine check --strict dist/*
 
+  pass:
+    needs: [check_dist]
+    runs-on: ubuntu-latest
+    steps:
+      - run: echo "All jobs passed"
+
```

Comment:
```suggestion
```

I like the consistency of having a pass, but it is identical to check_dist, so okay, just use that for now. Similar to ninja.